nexus-sdk-js/src/Project/index.ts

```diff
@@ -117,43 +130,96 @@ export default class Project {
 
   // The current API does not support pagination / filtering of views
   // This should be fixed when possible and converted to signature
-  // Promise<PaginatedList<ElasticSearchView>>
-  async listElasticSearchViews(): Promise<ElasticSearchView[]> {
+  // Promise<PaginatedList<(ElasticSearchView | SparqlView)>>
+  async listViews(): Promise<(ElasticSearchView | SparqlView)[]> {
```

Comment:
This is awesome! But it isn't consistent with the rest of the SDK. I'm going to use `Orgs` and `Projects` to illustrate it.
I'd move this code to the `utils.ts` file of `View`, like the code to retreive Projects https://github.com/BlueBrain/nexus-webapp-commons/blob/master/nexus-sdk-js/src/Project/utils.ts#L13

Then I would add a static method to the `View` class, like for `Project` https://github.com/BlueBrain/nexus-webapp-commons/blob/master/nexus-sdk-js/src/Project/index.ts#L57

Finally, I'd call this static method here, like `Organization` does https://github.com/BlueBrain/nexus-webapp-commons/blob/master/nexus-sdk-js/src/Organization/index.ts#L81

Happy to assist if needed!